api/data_explorer/controllers/facets_controller.py

```diff
@@ -1,15 +1,24 @@
-import connexion
-from datetime import date, datetime
-from typing import List, Dict
-from six import iteritems
-from ..util import deserialize_date, deserialize_datetime
+from datetime import date
+import json
+
+from data_explorer.models.facet import Facet
+from data_explorer.models.facet_value import FacetValue
+from ..dataset_faceted_search import DatasetFacetedSearch
 
 
 def facets_get():
     """
-    facets_get
     Returns facets.
 
-    :rtype: str
+    Returns:
+        List of Facet.
     """
-    return 'do some magic!'
+    search = DatasetFacetedSearch()
+    response = search.execute()
+    facets = []
+    for facet_name, values in response.facets.to_dict().iteritems():
+        facet_values = []
+        for value_name, count, _ in values:
```

Comment:
What's the third value here expected by unpacking ', _'? I don't see a third field in the API spec.